launch_param_builder/utils.py

```diff
@@ -52,31 +54,46 @@ class ParameterBuilderFileNotFoundError(KeyError):
 ]
 
 
+def render_template(template: Path, mappings: dict):
+    with template.open("r") as file:
+        jinja2_template = jinja2.Template(file.read())
+        jinja2_template.globals["radians"] = math.radians
```

Comment:
Is this what parses the !degrees and !radians in YAML?

Reply:
Yep, it adds a function to the jinja template